scripts/deploy_private_version.sh

```diff
@@ -106,6 +106,7 @@ else
     cp ${GENESISFILE} gen/${DEFAULTNETWORK}/genesis.json
 fi
 
+
```

Comment:
unless you're intentionally modifying this file, please revert this formatting.

Reply:
reverted